label_studio/data_import/api.py

```diff
@@ -523,3 +523,40 @@ def get(self, request, *args, **kwargs):
         response['X-Accel-Redirect'] = redirect
         response['Content-Disposition'] = 'attachment; filename="{}"'.format(filepath)
         return response
+
+
+class PresignStorageData(DownloadStorageData):
+    def get(self, request, *args, **kwargs):
+        """ Get export files list
+        """
+        request = self.request
+        task_id = kwargs.get("task_id")
+        fileuri = request.GET.get('uri')
+
+        if fileuri is None or task_id is None:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+        try:
+            task = Task.objects.get(pk=task_id)
+        except Task.DoesNotExist:
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+        project = task.project
+
+        if not project.has_permission(request.user):
```

Comment:
task.has_permissions() will work too, just FYI

Reply:
I saw that, but already we have the need to use project right after so I kept this for local consistency.